implementation/exporters/src/main/java/io/smallrye/opentelemetry/implementation/exporters/AbstractVertxExporterProvider.java

```diff
@@ -32,6 +38,10 @@ public abstract class AbstractVertxExporterProvider<T extends Marshaler> {
 
     private static final String MIMETYPE_PROTOBUF = "application/x-protobuf";
 
+    private static final String OTEL_EXPORTER_VERTX_CDI_QUALIFIER = "otel.exporter.vertx.cdi.qualifier";
```

Comment:
This should IMHO be called

```suggestion
    private static final String OTEL_EXPORTER_VERTX_CDI_QUALIFIER = "otel.exporter.vertx.cdi.identifier";
```

Because it is a value of the `@Identifier` qualifier, not a full qualifier (which would be very hard to denote).

Reply:
Done